tests/components/roborock/test_vacuum.py

```diff
@@ -38,12 +38,17 @@
 async def test_registry_entries(
     hass: HomeAssistant,
     entity_registry: er.EntityRegistry,
+    device_registry: dr.DeviceRegistry,
     bypass_api_fixture,
     setup_entry: MockConfigEntry,
 ) -> None:
     """Tests devices are registered in the entity registry."""
-    entry = entity_registry.async_get(ENTITY_ID)
-    assert entry.unique_id == DEVICE_ID
+    entity_entry = entity_registry.async_get(ENTITY_ID)
+    assert entity_entry.unique_id == DEVICE_ID
+
+    device_entry = device_registry.async_get(entity_entry.device_id)
+    assert device_entry is not None
+    assert device_entry.model_id == "roborock.vacuum.a27"
```

Comment:
That's a strange model id tbh

Reply:
it's the best model identifier per @Lash-L

Reply:
![image](https://github.com/user-attachments/assets/071a4dc9-4477-44d0-92b4-6e448b060535)

Yeah it's weird - but it is also how they display it in their app and in the code I have reverse engineered